library/archlinux

```diff
@@ -6,13 +6,13 @@ Maintainers: Santiago Torres-Arias <santiago@archlinux.org> (@SantiagoTorres),
              Justin Kromlinger <hashworks@archlinux.org> (@hashworks)
 GitRepo: https://github.com/archlinux/archlinux-docker.git
 
-Tags: latest, base, base-20201213.0.11146
-GitCommit: f70853f3a094e3c17b37c5d5aabda2ac713deb95
-GitFetch: refs/tags/v20201213.0.11146
+Tags: latest, base, base-20201220.0.11678
+GitCommit: f9d83c7a5c3cc3f14caaf45e63fd8cebfc3d2ddc
```

Comment:
I think there might've been a glitch somewhere in this publishing pipeline? https://github.com/archlinux/archlinux-docker/commit/f9d83c7a5c3cc3f14caaf45e63fd8cebfc3d2ddc :sweat_smile:

Reply:
Thanks, looks like our GitLab -> GitHub sync is dead. I'll have someone have a look.